instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/snsattributes_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelaws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sns"
+	"github.com/aws/smithy-go/middleware"
+	"github.com/stretchr/testify/assert"
+
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+)
+
+func TestSNSPublish(t *testing.T) {
```

Comment:
```suggestion
func TestSNSAttributeSetter(t *testing.T) {
```

Reply:
I think although right now `Publish` is the only operation supported by the setter, potentially in the future we may have a support for other operations, which may require a different setup for the tests and thus would rather be tested by a dedicated `Test<Operation>` function. I don't mind the renaming for how things are right now though

Reply:
Yeah, we should match what we currently have. If things change, the test should change too.